client/state/utils.js

```diff
@@ -375,3 +375,35 @@ export function combineReducersWithPersistence( reducers ) {
 	combined.hasSchema = validatedReducersHasSchema;
 	return combined;
 }
+
+/**
+ * Wraps a reducer such that it won't persist
+ * any state to the browser's local cache
+ *
+ * @example revent a simple reducer from persisting
+ * const age = ( state = 0, { type } ) =>
+ *   GROW === type
+ *     ? state + 1
+ *     : state
+ *
+ * export default combineReducers( {
+ *   age: withoutPersistence( age )
+ * } )
+ *
+ * @example preventing a large reducer from persisting
+ * const posts = withoutPersistence( keyedReducer( 'postId', post ) )
```

Comment:
I wonder if keyedReducer can call this directly if no deserializers are passed, since the example usage is an `opt-out` vs `opt-in`. The code otherwise looks spot on.